pandas/tests/io/json/test_ujson.py

```diff
@@ -600,23 +599,22 @@ def test_encode_list_long_conversion(self):
         )
 
     def test_encode_long_conversion(self):
-        long_input = 9223372036854775807
-        output = ujson.encode(long_input)
+        for long_input in [9223372036854775807, 18446744073709551615]:
```

Comment:
Instead of looping here will want to use `parametrize` - see test below for an example

Reply:
Done. See 61e51ac